candidates/templates/candidates/posts.html

```diff
@@ -25,7 +25,7 @@
                 <li>
                  <a href="{% url 'constituency' election=election.slug post_id=p.slug ignored_slug=p.short_label|slugify %}">{{ p.base.label }}</a>
 
-                 {% if p.candidates_locked %}
+                 {% if p.postextraelection_set.all.0.candidates_locked %}
```

Comment:
These comments might appear out of order, but just in case - I think `.get` would be better than `.all.0` here as I mentioned elsewhere.